tap_zendesk/streams.py

```diff
@@ -121,14 +122,17 @@ def raise_or_log_zenpy_apiexception(schema, stream, e):
     # it doesn't have access.
     if not isinstance(e, zenpy.lib.exception.APIException):
         raise ValueError("Called with a bad exception type") from e
-    if json.loads(e.args[0])['error']['message'] == "You do not have access to this page. Please contact the account owner of this help desk for further help.":
-        LOGGER.warning("The account credentials supplied do not have access to `%s` custom fields.",
-                       stream)
+
+    error = json.loads(e.args[0]).get('error')
+    if isinstance(error, dict):
+
+        if error.get('message', None) == "You do not have access to this page. Please contact the account owner of this help desk for further help.":
```

Comment:
Can we merge this if condition with the above one using `and`?

Reply:
Done

Reply:
> Can we merge this if condition with the above one using `and`?

Done